src/main/java/de/retest/recheck/ui/descriptors/idproviders/AbstractFirstNonNullRetestIdProvider.java

```diff
@@ -31,7 +30,7 @@ private String returnFirstNonBlank( final String... args ) {
 				return arg;
 			}
 		}
-		throw new IllegalStateException( "Should have at least one non-blank argument!" );
+		throw new IllegalStateException( "Type must be non-null and non-blank!" );
```

Comment:
I find this confusing, why type? I cannot find the term in the method. Probably this also should be an `IllegalArgumentException`?

Reply:
Sorry ... this doesn't belong to this branch...